README.md

```diff
@@ -72,3 +72,75 @@ the command below:
 ```
 bazel build --config=mycluster-ubuntu16-04 //...
 ```
+
+# Getting started
+
+This example aims to showcase a very simple build and test with remote execution using docker-compose as the deployment for Buildbarn. We will be compiling examples from the [abseill-hello](https://github.com/abseil/abseil-hello) project.
+
+## Recommended setup
+
+First clone the repo and start up a docker-compose example:
+```
+git clone https://github.com/buildbarn/bb-deployments.git
+cd bb-deployments/docker-compose
+./run.sh
+```
+
+You may see initially see an error message along the lines of:
+```
+worker-ubuntu16-04_1  | xxxx/xx/xx xx:xx:xx rpc error: code = Unavailable desc = Failed to synchronize with scheduler: all SubConns are in TransientFailure, latest connection error: connection error: desc = "transport: Error while dialing dial tcp xxx.xx.x.x:xxxx: connect: connection refused"
+```
+
+This is usually because container of the worker has started before the scheduler and so it cannot connect. After a second or so, this error message should stop.
+
+## Remote execution
+
+In a separate terminal, we will create a `.bazelrc` file and add a few extra options to it.
+```
```

Comment:
Does this part make some of the original documentation above superfluous? If so, be sure to remove parts of the old documentation.